code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/message/views/MessageFrame.kt

```diff
@@ -81,21 +82,23 @@ internal fun MessageFrame(
     // we want to ensure that the message is displayed relative to the activity's size.
     // This allows the message to be displayed correctly in split screen mode and other
     // multi-window modes.
+    val density = LocalDensity.current
     val contentView = currentActivity.findViewById<View>(android.R.id.content)
-    val contentHeightDp = with(LocalDensity.current) { contentView.height.toDp() }
-    val contentWidthDp = with(LocalDensity.current) { contentView.width.toDp() }
-    val heightDp = ((contentHeightDp * inAppMessageSettings.height) / 100)
-    val widthDp = ((contentWidthDp * inAppMessageSettings.width) / 100)
+    val contentHeightDp = with(density) { contentView.height.toDp() }
+    val contentWidthDp = with(density) { contentView.width.toDp() }
+    val heightDp = remember { mutableStateOf(((contentHeightDp * inAppMessageSettings.height) / 100)) }
+    val widthDp = remember { mutableStateOf(((contentWidthDp * inAppMessageSettings.width) / 100)) }
+    Log.debug(ServiceConstants.LOG_TAG, "MessageFrame", "Height: ${contentHeightDp.value} Width: ${contentWidthDp.value}")
```

Comment:
This will add a lot of noise to the logs due to recompositions. Let us not log from these Composables